docs/QUICKSTART.md

```diff
@@ -1,21 +1,44 @@
 # Quickstart #
 
-The CLI requires a few dependencies and C extensions that can be installed with
-`pip install securesystemslib[crypto,pynacl]`.
+In this quickstart tutorial, we'll use the basic TUF command-line interface
+(CLI), which includes the `repo.py` script and the `client.py` script, to set
+up a repository with an update and metadata about that update, then download
+and verify that update as a client.
+
+Unlike the underlying TUF modules that the CLI uses, the CLI itself is a bit
+bare-bones.  Using the CLI is the easiest way to familiarize yourself with
+how TUF works, however.  It will serve as a very basic update system and use
 
 ----
-The following is a basic workflow in four steps:
 
-**Step (1)** - Initialize a repo.  The `tufrepo`, `tufkeystore`, and
-`tufclient` directories are created in the current working directory.
+**Step (0)** - Make sure TUF is installed
+
+See the [installation instructions for TUF](docs/INSTALLATION.rst).
```

Comment:
We are in the same directory `INSTALLATION.rst`. Remove `docs` form relative path or use absolute path.

Reply:
Also, would it be bad to just provide the snippet here? So that quickstarters don't have to go through INSTALLATION.rst?
```
pip install tuf securesystemslib[crypto,pynacl]
```

Reply:
They do need to go through the installation instructions to install TUF.  Prior text was a bit misleading in that it suggested that they only needed to run the ssl install.  In any case, that step is listed two lines down, to emphasize it, since it would otherwise be optional.

Reply:
But isn't the line I'm suggesting all that's required for the quick start?

Reply:
No, you need TUF installed.

Reply:
Hopefully the newer text is less likely to lead to that conclusion.

Reply:
😞 I kept reading your suggested line here as the same as the existing line two lines below, and as the last line in INSTALLATION.md  -- I missed that sneaky "tuf" inclusion.  Sorry. 😄

I also managed to miss that the "external dependencies" section I was assuming that the [installation instructions](INSTALLATION.md) still included is now only in [`CONTRIBUTORS.rst`](https://github.com/theupdateframework/tuf/blob/e82ce352b42cd7012571657fe4a1a70d63368751/docs/CONTRIBUTORS.rst#external-dependencies).  That's where lines like `apt-get install build-essential libssl-dev libffi-dev python-dev` live....  I'm not sure why that makes sense.  I need to try installation out without all those on a vm.

Reply:
Simplified.  Installing pip produces everything you need already on a basic Ubuntu platform, so I didn't delve into python-dev or libssl-dev, etc.  Just provided the single command in the QUICKSTART.

Reply:
>  python-dev or libssl-dev

To contextualize, you need this to build the native support for the openssl backend. I think it's better to keep this from becoming a transient dependency as that may change  at some point in the future...

Reply:
I'm not clear on your suggestion, sorry:  are you saying that we should avoid _implicitly_ requiring that `libssl-dev` or similar packages be installed, or that we should avoid advising it explicitly since it may not be necessary?  (Or something else?)

Reply:
oh, the opposite. I understood that you were advising to remove these deps because something else is pulling them in. I'd rather have them explicitly just in case this is not the default in the future.

Reply:
Currently, those requirements are now only in the collaborator docs, not in the installation docs.  I was wondering if I should move them back into the installation docs.  You're suggesting that we include them, which is good to hear, so I'll pull them back in to the natural place to put them (the installation instructions).  I'll frame them as as-necessary installations, I guess, to try to keep things simple.

That's beyond the scope of this PR, so I'll just do that in the next one, but for this PR, I'll split the difference in `QUICKSTART.md` in anticipation of that change....